jumpstarter/states.py

```diff
@@ -34,6 +34,34 @@
 NestedState.separator = "↦"
 
 
+class Transition(str, Enum):
+    def _generate_next_value_(name: str, *args) -> str:
+        name = name.replace("__", NestedState.separator)
+        return name.lower()
+
+    INITIALIZE = auto()
```

Comment:
We use lowercase in Enums and I believe that's the right way to do it.
There's no PEP rule though. 
https://mail.python.org/pipermail/python-ideas/2016-September/042340.html

Reply:
I think that email confirms the way that I typically think of enums, and the way I have seen them used elsewhere. IMO, they are constants. There also is the added benefit that being uppercase, syntax highlighters show them differently making them easier to spot and recognize.

![image](https://user-images.githubusercontent.com/166269/111137597-a6c5eb80-8587-11eb-968b-7639a1845727.png)

versus

![image](https://user-images.githubusercontent.com/166269/111137693-c52be700-8587-11eb-9db4-0b6bd447bc49.png)

---

I would therefore personally advocate that they all be made uppercase. But, since they are not, consistency mandates lowercase, I agree with you there.